subsys/bluetooth/controller/hci/hci.c

```diff
@@ -2592,10 +2602,19 @@ static void le_adv_ext_report(struct pdu_data *pdu_data, u8_t *b,
 {
 	struct pdu_adv *adv = (void *)pdu_data;
 	s8_t rssi;
+	u8_t *extra;
 
+#if defined(CONFIG_BT_LL_SW_SPLIT)
+	struct node_rx_pdu *node_rx;
+
+	node_rx = (struct node_rx_pdu *)b;
+	extra = (u8_t *) &(node_rx->hdr.rx_ftr.param);
+#else
+	extra = (u8_t *) &b[offsetof(struct radio_pdu_node_rx, pdu_data) +
```

Comment:
no space in between ```(u8_t *) &(```   and in all other such occurences.

Reply:
Thanks for pointing this out, now fixed